What's new in 8.17 (#6286)
elastic/security-docs: docs/whats-new.asciidoc

{security-guide-all}/7.9/whats-new.html[7.9]

// NOTE: The notable-highlights tagged regions are re-used in the Installation and Upgrade Guide. Full URL links are required in tagged regions.
// tag::notable-highlights[]

[float]
== Detection rules and alerts enhancements

[float]
=== LogsDB index mode with detection rules and alerts

> The {ref}/logs-data-stream.html[logsDB index mode] allows you to store log data more efficiently. If you're considering using it, refer to <TBD> to learn how it can impact your rules and alerts. This feature requires the <TBD>.

Reviewer: https://github.com/elastic/security-docs/pull/6272 needs to be merged for this reference to properly resolve. 

```suggestion
The {ref}/logs-data-stream.html[logsDB index mode] allows you to store log data more efficiently. If you're considering using it, refer to <<detections-logsdb-index-mode-impact>> to learn how it can impact your rules and alerts. 
```

// some information to be added

[float]
=== Suppress alerts for EQL sequence rules

> Alert suppression now supports the EQL sequence rule type. You can use it to reduce the number of repeated or duplicate detection alerts generated from EQL sequence rules.

Reviewer: ```suggestion
<<alert-suppression,Alert suppression>> now supports the EQL sequence rule type. You can use it to reduce the number of repeated or duplicate detection alerts generated from EQL sequence rules.
```

> The {ref}/logs-data-stream.html[logsdb index mode] allows you to store log data more efficiently. If you're considering using it, refer to Using logsdb index mode with {elastic-sec} to learn how it can impact your rules and alerts.

Reviewer: Re-adding the link so you can merge this comment after I merge https://github.com/elastic/security-docs/pull/6272.

```suggestion
The {ref}/logs-data-stream.html[logsdb index mode] allows you to store log data more efficiently. If you're considering using it, refer to <<detections-logsdb-index-mode-impact,Using logsdb index mode with {elastic-sec}>> to learn how it can impact your rules and alerts.

NOTE: To use the {ref}/mapping-source-field.html#synthetic-source[synthetic `_source`] feature, you must have the appropriate subscription. Refer to the subscription page for https://www.elastic.co/subscriptions/cloud[Elastic Cloud] and {subscriptions}[Elastic Stack/self-managed] for the breakdown of available features and their associated subscription tiers.
```

{security-guide}/alert-suppression.html[Alert suppression] now supports the EQL sequence rule type. You can use it to reduce the number of repeated or duplicate detection alerts generated from EQL sequence rules.

[float]
== Signature option available for macOS trusted applications conditions

When adding a {security-guide}/trusted-apps-ov.html[trusted application] for macOS, you can now specify conditions based on the application's digital signer—previously only available on Windows.

[float]

> == Case action is generally available

Reviewer: ```suggestion
== Cases action is generally available
```

> The {kibana-ref}/cases-action-type.html[Case action] feature, first introduced in 8.14, is moving from technical preview to general availability. Use Case action to automatically create cases from rules.

Reviewer: Minor correction, based on the naming used in the [feature docs](https://www.elastic.co/guide/en/kibana/current/cases-action-type.html).
```suggestion
The {kibana-ref}/cases-action-type.html[Cases action] feature, first introduced in 8.14, is moving from technical preview to general availability. Use this action to automatically create cases from rules.
```
